fluentd/lib/fluent-plugin-collected/fluent-plugin-collected.gemspec

```diff
@@ -13,12 +13,10 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "fluentd", ">= 0.14.20", "< 2"
-  spec.add_dependency "prometheus-client", "< 0.10"
+  spec.add_dependency "fluentd", ">= 0.14.5", "< 2"
```

Comment:
I don't know how this bubbles up to the vendoring but we should ensure we only use 1.14.5